### ISSUE TITLE
Add baises to neural network

### ISSUE THREAD

**Issue by AxlLind** [AxlLind/Number-Recognition#1]
Neural network currently only uses weigths. Biases should be added. Not 100% sure if this will increase the percentage of correctly classified.

Main issue will be figuring out the partial derivative with respect to the bais which is used in gradient descent.

**Comment by AxlLind** [comment]
After some more research it seems the MNIST dataset doesn't really benefit from the extra degree of freedom that the bias brings.
